### ISSUE TITLE
Browse category and Feature page thumbnail redesign

### ISSUE THREAD

**Issue by alundgard** [sul-dlss/exhibits#2632]
_Implement redesigned Browse category and Feature page thumbnail for improved accessibility._

The Stanford Office of Digital Accessibility (SODA) confirmed the following accessibility issues with the current thumbnail design. Text appearing overlaid on background images must comply with WCAG color contrast guidelines. The thumbnail images are specified by exhibit creators, and currently do not always satisfy the color contrast ratio with the overlaid text.

Browse category thumbnail example: https://exhibits.stanford.edu/martin-wong/browse
Feature page thumbnail example: https://exhibits.stanford.edu/virtual-tribunals

The thumbnail redesign specification is available in the “Exhibits Redesign 2024” Figma File:
https://www.figma.com/design/0LTbMiP5xdL6T94nmlKDci/Exhibits-Redesign-2024?node-id=130-6586
![Image](https://github.com/user-attachments/assets/b16f5950-cf1b-47a2-8e3f-43d37f296a79)

**Comment by corylown** [comment]
This may end up being work in Spotlight or both Spotlight and Exhibits depending on where the current layout and styles are defined.

**Comment by dnoneill** [comment]
Waiting until @alundgard can review the look to make sure it is right. https://github.com/projectblacklight/spotlight/pull/3307
